.github/ISSUE_TEMPLATE/2-feature_request.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Feature request
+about: Suggest an idea for XiangShan project.
+title: ''
+labels: 'feature request'
+assignees: ''
+
+---
+
+#### Before start
+PLEASE MAKE SURE you have done these: 
+- [x] (Select what you have done like this)
+- [ ] I have read the [RISC-V ISA Manual](https://github.com/riscv/riscv-isa-manual).
+- [ ] I have read the [XiangShan Documents](https://xiangshan-doc.readthedocs.io/zh_CN/latest).
+- [ ] I have searched the previous issues and did not find anything relevant.
+- [ ] I have reviewed the commit messages from the relevant commit history.
+
+#### Is your feature request related to a problem? Please describe.
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+#### Describe the solution you'd like
+A clear and concise description of what you want to happen.
```

Comment:
A clear and concise description of what you want.

No need to add "to happen?"